### DISCUSSION TITLE
Is LTE really based on bootstrap 5?

### DISCUSSION THREAD

**Discussion by krulis-martin** [ColorlibHQ/AdminLTE#5294] [General]
Hi,
I apologize for my confusion, but is the current AdminLTE 3.2.0 based on Bootstrap 5? The thing is I did not find any mention of this upgrade in the changelogs (the latest upgrade I found was to Bootstrap 4.6) and if I checked correctly, the [AdminLTE live demo](https://adminlte.io/) also uses Bootstrap 4.6 (at least the JS plugin says so). Could you clarify, please? :)

**Comment by REJack** [comment]
3.2.0 is based on Bootstrap 4, AdminLTE v4(master branch) is based on Bootstrap 5.

**Reply by krulis-martin** [reply]
Thanks for clarifying! I would humbly suggest adding this information to the readme as well (since now it appears as if the current version is based on Bootstrap 5).
